sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -88,12 +90,25 @@ private[spark] object HiveUtils extends Logging {
       |   <code>${builtinHiveVersion}</code> or not defined.
       | 2. "maven"
       |   Use Hive jars of specified version downloaded from Maven repositories.
-      | 3. A classpath in the standard format for both Hive and Hadoop.
+      | 3. "path"
+      |   Use Hive jars configured by `spark.sql.hive.metastore.jars.path` in comma separated format
```

Comment:
`${HIVE_METASTORE_JARS_PATH.key}`

Reply:
> `${HIVE_METASTORE_JARS_PATH.key}`

We can't refer to each other between `HIVE_METASTORE_JARS_PATH` and `HIVE_METASTORE_JARS`